### PR TITLE
DEV-2806: Update layout for contributions export

### PR DESCRIPTION
**Pull request by GuiMend**
Please fill out each section even if it's just with "N/A"

#### Plan? (if this draft/incomplete indicate what you intend to do and how)
n/a

#### What's this PR do?
Updates the email template that is sent when user's asks for contributions export.

#### Why are we doing this? How does it help us?
Email template was just text, so we needed to update the design.

#### How should this be manually tested? Please include detailed step-by-step instructions.
- Login
- Access contributions page
- Click Export button
- Check that email design matches figma

#### Did this PR make changes to the user interface that may require changes to the user-facing docs?
Yes

#### Have automated unit tests been added? If not, why?
Yes

#### How should this change be communicated to end users?
n/a

#### Are there any smells or added technical debt to note?
no

#### Has this been documented? If so, where?
no

#### What are the relevant tickets? Add a link to any relevant ones.
https://news-revenue-hub.atlassian.net/browse/DEV-2806

#### Do any changes need to be made before deployment to production (adding environment variables, for example)?
no

#### Are there next steps? If so, what? Have tickets been opened for them? List next-step tickets here:
no

[DEV-2806]: https://news-revenue-hub.atlassian.net/browse/DEV-2806?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/emails/templates/nrh-contribution-csv-email-body.html [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{% extends "nrh-user-base-template.html" %}
+
+{% block content %}
+<p class="content-header">Your Export is Ready!</p>
```
Comment (benjaminEwhite) [inline apps/emails/templates/nrh-contribution-csv-email-body.html:4]:
```suggestion
<p class="content-header">Your export is ready!</p>
```

File: apps/emails/templates/nrh-contribution-csv-email-body.html [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+<p class="content-body">Download the attached csv to access your exported contributions data.</p>
```
Comment (benjaminEwhite) [inline apps/emails/templates/nrh-contribution-csv-email-body.html:5]:
```suggestion
<p class="content-body">Download the attached CSV file to access your exported contributions data.</p>
```

**Review by benjaminEwhite — CHANGES_REQUESTED** [review]
Overall looks good. Couple of tiny requests re: copy, but those may need to be confirmed with Mariclare and/or Rachel.

File: apps/emails/templates/nrh-contribution-csv-email-body.html (hunk 1 above)
Comment (GuiMend) [reply apps/emails/templates/nrh-contribution-csv-email-body.html:4]: This one we want to keep it like it is. 👍

**Review by benjaminEwhite — APPROVED** [review]

**Review by x110dc — APPROVED** [review]

**Pull request merged**